app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt

```diff
@@ -23,16 +25,17 @@ class OverdueListAdapter : ListAdapter<OverdueListItem, OverdueListViewHolder>(O
 }
 
 data class OverdueListItem(
+    val appointmentId: UUID,
     val name: String,
     val gender: String,
     val age: Int,
     val bpSystolic: Int,
     val bpDiastolic: Int,
-    val bpDate: String,
+    val bpDaysAgo: Int,
     val overdueDays: Int
 )
 
-class OverdueListViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+class OverdueListViewHolder(itemView: View, val context: Context) : RecyclerView.ViewHolder(itemView) {
```

Comment:
This VH can get the context from `itemView` directly. Do you think we can remove this extra param?